src/layouts/Default.vue

```diff
@@ -1,18 +1,31 @@
 <template>
-  <div class="layout">
-    <header class="header">
-      <strong>
-        <g-link to="/">{{ $static.metadata.siteName }}</g-link>
-      </strong>
-      <nav class="nav">
-        <g-link class="nav__link" to="/">Home</g-link>
-        <g-link class="nav__link" to="/about/">About</g-link>
-        <g-link class="nav__link" to="/faq/">FAQ</g-link>
-        <g-link class="nav__link" to="/blog/">Blog</g-link>
-      </nav>
-    </header>
-    <slot/>
-  </div>
+    <div class="layout">
+        <header class="masthead">
+            <b-navbar class="container justify-content-center" toggleable="lg">
+                <b-navbar-brand href="#">
+                    <img src="assets/images/logos/colour_text.png" class="d-inline-block align-top" alt="GMN Logo">
```

Comment:
@rahmot could you put the logo correctly? probably like in the hub: just the logo without the text, in `src` folder, named as `favicon,png` with a smaller resolution

Reply:
Done, thanks!